cmake/MSCCLPP.cmake

```diff
@@ -46,7 +46,7 @@ if(ENABLE_MSCCLPP)
     
         download_project(PROJ                mscclpp_nccl
                          GIT_REPOSITORY      https://github.com/microsoft/mscclpp.git
-                         GIT_TAG             b1b9d0626cfa40319c18c05f8c16650568395c29
+                         GIT_TAG             8c6fb429e92e07acb82c0fdcdab44854fc63aa68
```

Comment:
Has the bf16 PR been merged to MSCCLPP `main` branch?

Reply:
Yes